lib/tasks/active_fedora_dev.rake

```diff
@@ -51,10 +51,23 @@ namespace :active_fedora do
   desc "CI build"
   task :ci do
     ENV['environment'] = "test"
-    # Rake::Task["active_fedora:configure_jetty"].invoke
-    # jetty_params = Jettywrapper.load_config
-    solr_params = { port: 8983, verbose: true, managed: true }
-    fcrepo_params = { port: 8984, verbose: true, managed: true }
+    solr_params = { port: 8985, verbose: true, managed: true }
+    fcrepo_params = { port: 8986, verbose: true, managed: true,
+                      java_options: ['-Dfcrepo.log.http.api=WARN',
+                                     # To avoid "WARN: The namespace of predicate:
+                                     # info:fedora/fedora-system:def/relations-external#isPartOf
+                                     # was possibly misinterpreted as:
+                                     # info:fedora/fedora-system:def/relations-external#."
+                                     '-Dfcrepo.log.kernel=ERROR',
+                                     # To prevent a second instance (dev/test) of Fedora
+                                     # from having a port collision when starting up
+                                     # ActiveMQ, set a custom port for this test instance
+                                     '-Dfcrepo.dynamic.jms.port=61617',
```

Comment:
https://github.com/cbeer/fcrepo_wrapper/pull/3 ought to help with these overrides.